src/app/containers/CpsRelatedContent/index.jsx

```diff
@@ -43,18 +42,26 @@ const formatItem = (item, env) => {
   return assocPath(['locators', 'assetUri'], `${baseUri}${uriSuffix}`, item);
 };
 
-const CpsRelatedContent = ({ content }) => {
+const CpsRelatedContent = ({ content, enableGridWrapper }) => {
   const { script, service, dir, translations } = useContext(ServiceContext);
   const { env } = useContext(RequestContext);
+  const ConditionalWrapper = ({ condition, wrapper, children }) =>
+    condition ? wrapper(children) : children;
+  const a11yAttributes = {
+    as: 'section',
+    role: 'region',
+    'aria-labelledby': 'related-content-heading',
+  };
   if (!content.length) return null;
 
   return (
-    <GhostGrid
-      as="section"
-      role="region"
-      aria-labelledby="related-content-heading"
+    <ConditionalWrapper
+      condition={enableGridWrapper}
+      wrapper={children => (
+        <GhostGrid {...a11yAttributes}>{children}</GhostGrid>
+      )}
     >
-      <Wrapper>
+      <Wrapper {...(enableGridWrapper || a11yAttributes)}>
```

Comment:
I find the conditions here quite difficult to follow. I'd suggest doing the following. It's more lines of code but hopefully easier to understand at a quick glance.

```
  const GridWrapper = ({ children }) => (
    <GhostGrid {...a11yAttributes}>
      <Wrapper>{children}</Wrapper>
    </GhostGrid>
  );
  GridWrapper.propTypes = {
    children: node.isRequired,
  };

  const NoGridWrapper = ({ children }) => (
    <Wrapper {...a11yAttributes}>{children}</Wrapper>
  );

  NoGridWrapper.propTypes = {
    children: node.isRequired,
  };

  const RelatedContentWrapper = enableGridWrapper ? GridWrapper : NoGridWrapper;


  return (
    <RelatedContentWrapper>
      <StyledSectionLabel
        script={script}
        service={service}
        dir={dir}
        labelId="related-content-heading"
      >
        {translations.relatedContent}
      </StyledSectionLabel>

      <StoryPromoUl>
      ......
      </StoryPromoUl>
    </RelatedContentWrapper>
  );
```

Reply:
Just got off a call with Simon and discussed your solution. Would you be happy with this?
```
const RelatedContentWrapper = ({ children }) =>
    enableGridWrapper ? (
      <GhostGrid {...a11yAttributes}>
        <Wrapper>{children}</Wrapper>
      </GhostGrid>
    ) : (
      <Wrapper {...a11yAttributes}>{children}</Wrapper>
    );
  RelatedContentWrapper.propTypes = {
    children: node.isRequired,
  };

return (
    <RelatedContentWrapper>
      <StyledSectionLabel
        script={script}
        service={service}
        dir={dir}
        labelId="related-content-heading"
      >
        {translations.relatedContent}
      </StyledSectionLabel>

      <StoryPromoUl>
      .......
      </StoryPromoUl>
    </RelatedContentWrapper>
  );
```

Reply:
Yes, that's much tidier. Thanks.